src/cleanlab_tlm/utils/rag.py

```diff
@@ -131,6 +132,18 @@ def __init__(
 
             self._evals = evals
 
+        self._validate_disable_trustworthiness_option(options)
+
+    def _validate_disable_trustworthiness_option(self, options: Optional[TLMOptions]) -> None:
+        """Validate that when trustworthiness scoring is disabled, alternative evaluations are available."""
+        disable_trustworthiness = options and options.get("disable_trustworthiness", False)
+
+        if disable_trustworthiness and not self._evals:
+            raise ValidationError(
+                f"When disable_trustworthiness=True in {self.__class__.__name__}, at least one evaluation must be provided. "
+                "Either provide evaluations via the 'evals' parameter or set disable_trustworthiness=False."
+            )
+
```

Comment:
The `validate_tlm_options` helper is called in `BaseTLM`, so you might actually want to handle this in that function instead of doing it separately here (gets confusing / diverging logic)